.vscode/settings.json

```diff
@@ -0,0 +1,8 @@
+{
+    "markdownlint.config": {
+        "MD028": false,
+        "MD025": {
+            "front_matter_title": ""
+        }
+    }
+}
```

Comment:
This file shouldn't be checked in. Can you help to see why this file is not ignored by git?

Reply:
Right.
Yes, `.vscode` directory was not in the ignored directories. I have added it now.